.mvn/modernizer/violations.xml

```diff
@@ -143,7 +143,14 @@
     <violation>
         <name>org/apache/hadoop/conf/Configuration."&lt;init&gt;":()V</name>
         <version>1.1</version>
-        <comment>Prefer new Configuration(false)</comment>
+        <!-- retain mention of `new Configuration(false)` as it describes why this constructor is discouraged for the first place -->
+        <comment>Prefer new Configuration(false), actually use ConfigurationInstantiator.newEmptyConfiguration()</comment>
```

Comment:
This sounds strange. How about
```xml
<comment>Prefer ConfigurationInstantiator.newEmptyConfiguration()</comment>
```
And add a comment here explaining why

Reply:
good idea, changing